### PR TITLE
Add script to create user and developer documentation bundle

### PR DESCRIPTION
**Pull request by brainwane**
To help us inform our stakeholders better, it's nice to have a ZIPped
archive of our user manual, developer documentation, and so on. This
script creates a temporary directory, builds PDFs of our prose
documents for users, administrators, and developers, builds the HTML
documentation for our API, downloads PDFs and PNG images of
architectural and schema diagrams, and puts together a ZIP file with a
PDF index page with clickable links to aid browsing.

~~~
$ cd scripts
$ ./make-zip-of-documents.sh
~~~

To run this successfully, you'll need some Debian packages installed;
see `docs/userhelp/README.mdwn`.

Towards #458.

### CODE REVIEW

File: scripts/make-zip-of-documents.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+# Grab architecture diagrams
+
+cd $DOCBUNDLEDIR
+wget -O 5_architecture-for-stakeholders-diagram.pdf \
+     "https://groups.google.com/group/psm-dev/attach/9a584f735379f/psm-architecture-for-stakeholders.pdf?part=0.2&authuser=0"
+wget -O 6_enrollment-process-diagram.png "https://groups.google.com/group/psm-dev/attach/9a584f735379f/EnrollmentProcess.png?part=0.1&authuser=0"
+wget -O 7_screening-process-diagram.png "https://groups.google.com/group/psm-dev/attach/9a584f735379f/screening-structural-diagram.png?part=0.3&authuser=0"
+wget -O 8_schema-compact-diagram.png https://user-images.githubusercontent.com/842790/32683739-9e56a920-c64b-11e7-9c0d-e05f5031a8d8.png
+wget -O 9_schema-hierarchical-diagram.png https://user-images.githubusercontent.com/842790/32683738-9e4090ea-c64b-11e7-9d10-c51944986bc3.png
```
Comment (cecilia-donnelly) [inline scripts/make-zip-of-documents.sh]: What happens if these `wget` calls fail?  Ideally the script would still run even without them.

**Comment by brainwane** [comment]
Because of the link problem, I'm going to switch the ToC to be HTML rather than a PDF.

**Comment by brainwane** [comment]
I've now integrated `ICD.md`.

**Comment by brainwane** [comment]
@cecilia-donnelly I am as a dog-at-the-computer when it comes to CSS and so I figured it was better for me to throw together a stylesheet based on the Sphinx stuff I'm used to than to attempt to learn the fiddlinesses of one of the PSM stylesheets, even though that required I find someplace to keep this `style.css` file and right now it seems ok-enough to put it in `scripts`. Happy to defer to someone else if they want to swoop in and revamp all the CSS-related stuff this script uses!

**Pull request merged**